sdk/cosmos/azure-cosmos-benchmark/ctl/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y openjdk-8-jdk net-tools vim
+COPY ./run_benchmark.sh /usr/app/
+COPY ./azure-cosmos-benchmark-jar-with-dependencies.jar /usr/app/
```

Comment:
Docket image nees to be build from benchamrk target folder right?

Reply:
Building code and build image are two separate task, so we have copy task which bring the jar and docker config files in one folder before we build docker . Please refer the CI sent offline